packs/actions/quick-alchemy.json

```diff
@@ -18,7 +18,14 @@
             "remaster": true,
             "title": "Pathfinder Player Core 2"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "CraftingAbility",
+                "priority": 1,
```

Comment:
?

Reply:
It doesn't rely on anything and its intent is to be expanded. This is the one on the ability itself.

Reply:
I mean the priority. We should almost never use 1

Reply:
Is the idea that it prevents someone from going first layter? It absolutely should always go first and I can't see a scenario where it does not, but if it makes you feel more confident, I can set it to 8, so it goes below all default priorities in the system and future proofs crafting ability being changed to default to 9 instead of 19 (which is something I'm still considering).

Reply:
It has a default of 100.